nengo_viz/static/viz_value.js

```diff
@@ -22,28 +23,50 @@ VIZ.Value = function(args) {
     this.svg = d3.select(this.div).append('svg')
         .attr('width', '100%')
         .attr('height', '100%');
-        
+
+    //Add the title 
+    this.svg.append("text")
+        .text(args.id)
```

Comment:
(and the same thing should happen for all Components, for that matter, so it might be best if this was handled in ```viz.js``` on the client side (like id and width and height and x and y are right now)